Source/WebKit/UIProcess/Cocoa/VideoPresentationManagerProxy.mm

```diff
@@ -775,8 +778,8 @@ - (BOOL)prefersStatusBarHidden
         [playerLayer layoutIfNeeded];
     }
 
-    if (m_page)
-        m_page->legacyMainFrameProcess().send(Messages::VideoPresentationManager::EnsureUpdatedVideoDimensions(contextId, nativeSize), m_page->webPageIDInMainFrameProcess());
+    if (RefPtr page = m_page.get())
```

Comment:
The static analyzer didn't say the use of `m_page` was unsafe in this and some other locations (just said `legacyMainFrameProcess` is unsafe). But other instances in this file do create a Ref pointer on the stack in this way--as does the other file in this PR. So I thought I'd do it in all the other places in this file anyways for consistency. Is this ok?